client/components/App.jsx

```diff
@@ -2,9 +2,12 @@ import React from 'react'
 
 import LoadSubreddit from './LoadSubreddit'
 import SubredditContainer from '../containers/SubredditContainer'
+import Home from './Home'
 
 const App = () => (
   <div className='app-container'>
+    <Home />
+    <p></p>
```

Comment:
It looks like this p tag is only used for spacing - it's better to use css instead of empty elements if you want space between things.